test-requirements.txt

```diff
@@ -2,10 +2,9 @@
 azure-storage
 google-cloud-storage
 h2o
-# TODO: don't pin boto version once https://github.com/spulec/moto/issues/1793 is addressed
```

Comment:
This issue has been closed via a PR that was merged and incorporated into Moto release 1.3.7